torchgen/model.py

```diff
@@ -656,8 +660,9 @@ def from_yaml(
                     )
                     if (
                         dispatch_key is DispatchKey.CompositeImplicitAutograd
-                        and v == cpp.name(func)
-                    ):
+                        or dispatch_key
```

Comment:
could be `dispatch_key in {DispatchKey.CompositeImplicitAutograd, DispatchKey.CompositeImplicitAutogradNestedTensor}`